lfs/config.go

```diff
@@ -327,28 +328,22 @@ func (c *Configuration) loadGitConfig() bool {
 
 	c.gitConfig = make(map[string]string)
 	c.extensions = make(map[string]Extension)
+	uniqRemotes := make(map[string]bool)
 
-	listOutput, err := git.Config.List()
-	if err != nil {
-		panic(fmt.Errorf("Error listing git config: %s", err))
-	}
+	configFiles := []string{
+		filepath.Join(LocalWorkingDir, ".lfsconfig"),
 
-	configFile := filepath.Join(LocalWorkingDir, ".gitconfig")
-	fileOutput, err := git.Config.ListFromFile(configFile)
-	if err != nil {
-		panic(fmt.Errorf("Error listing git config from %s: %s", configFile, err))
+		// TODO: remove .gitconfig support for Git LFS v2.0
```

Comment:
Add the issue # tracking this work so that when we complete the work we know to remove the TODO.